src/autoscaler/metricscollector/cmd/metricscollector/main.go

```diff
@@ -32,14 +32,13 @@ func main() {
 		os.Exit(1)
 	}
 
-	var conf *config.Config
-
 	configFile, err := os.Open(path)
 	if err != nil {
 		fmt.Fprintf(os.Stdout, "failed to open config file '%s' : %s\n", path, err.Error())
 		os.Exit(1)
 	}
 
+	var conf *config.Config
 	conf, err = config.LoadConfig(configFile)
```

Comment:
yes right.  I experienced a  defect where I misused := in a If/for which results in a new local variable created not reused.  Just want to make it more clearer where the variable is claimed.